blocks/init/src/Blocks/components/lists/components/lists-editor.js

```diff
@@ -1,8 +1,8 @@
 import React, { useMemo } from 'react';
-import classnames from 'classnames';
 import { __ } from '@wordpress/i18n';
 import { RichText } from '@wordpress/block-editor';
-import { outputCssVariables, getUnique, selector, checkAttr, getAttrKey, getOption } from '@eightshift/frontend-libs/scripts';
+import { outputCssVariables, getUnique, selector, checkAttr, getAttrKey, getOption, classnames
+ } from '@eightshift/frontend-libs/scripts';
```

Comment:
looks a bit strange maybe to put each in a separate row?

```suggestion
import {
outputCssVariables,
getUnique,
selector,
checkAttr,
getAttrKey,
getOption,
classnames,
} from '@eightshift/frontend-libs/scripts';
```

Reply:
@volfkarlo  But you did add the helper in `scripts/index.js`, so there is no need for me to do anything about it?

Reply:
Did I? 😂 

If it works awesome 😄